docs/experience/memory-usage.md

```diff
@@ -0,0 +1,39 @@
+title: Monitoring Browser Memory Usage
+description: Gathering information about web page memory usage
+
+Browsers manage the memory used by web pages automatically without user interaction. Sematext Experience allows monitoring the memory usage for your web pages to identify issues, regressions, and give you an estimate of the memory needed by your applications. 
+
+<img
+  class="content-modal-image"
+  alt="Memory Usage Overview"
+  title="Memory Usage Overview"
+  src="../../images/experience/memory/overview.png"
+/>
+
+## Use Cases
+There are many use cases where having insight into the browser's memory usage can be crucial to identify and fix problems as soon as possible. Those include:
+
+ * Identifying places in your application resulting in high memory usage
+ * A/B testing of memory usage between different application versions
+ * Measuring memory impact of new features or optimizations done to the application code
+ * Statistical analysis of memory usage across the web application
+
+<img
+  class="content-modal-image"
+  alt="Memory Usage Top Sessions"
+  title="Memory Usage Top Sessions"
+  src="../../images/experience/memory/topsessions.png"
+/>
+
+## Measurements
+Sematext Experience [Browser SDK](https://sematext.com/docs/agents/browser/) uses the [Memory Measure API](https://wicg.github.io/performance-measure-memory/) to measure the memory usage when your web application is loaded and continues to measure the memory throughout the user session. This allows for memory measurements across pages and sessions. It even shows the per-session memory usage that the user of your applications was experiencing. 
```

Comment:
to measure the memory usage => to measure browser memory usage
that the user of your applications was experiencing => that individual users of your applications are experiencing.

Reply:
Adjusted. Thanks.